docs/data/joy/components/typography/TypographyBasics.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import Card from '@mui/joy/Card';
+import Typography from '@mui/joy/Typography';
+
+export default function TypographyBasics() {
+  return (
+    <Card variant="outlined" sx={{ width: 400 }}>
+      <Typography level="h1" fontSize="lg" sx={{ mb: 0.5 }}>
+        National Parks
+      </Typography>
+      <Typography level="h3" fontSize="md">
```

Comment:
Let's make this an h2 so it can directly follow the h1 above.

```suggestion
      <Typography level="h2" fontSize="md">
```